update: RPi page (#134)
Ultramarine-Linux/wiki: src/content/docs/en/anywhere/rpi.mdx

## Selecting a Drive

> The drive you use will affect the experience you will have on Ultramarine.

Reviewer: ```suggestion
The drive you use will affect the experience you'll have on Ultramarine.
```

[Raspberry Pi Imager instead](/en/anywhere/rpi#raspberry-pi-imager).
First, you will need to decompress it.

- On Linux, run `unzstd FILENAME.img.zst`, you may need to install the `zstd` package
- On Windows 11 or ChromeOS, zstd is natively supported. Decompress like you would a `.zip` file.
- On Windows 10, use 7zip or WinRaR
- On macOS, install `zstd` from [Homebrew](https://brew.sh/) (`brew install zstd`), then `unzstd FILENAME.img.zst`.

Grab a USB stick or SD Card! Use a flasher program to flash Ultramarine to your storage device. [balenaEtcher](https://www.balena.io/etcher/) is an easy cross-platform choice.

![](/assets/balenaetcher.png)

Simply select the .img file, the device you're using, and click flash.

You can also use the Disks application on Budgie, Xfce, and GNOME editions.

## Raspberry Pi Imager

> - Install [Raspberry Pi Imager](https://www.raspberrypi.com/software/). Note that this should be packaged in your distribution, and it's also available on other operating systems, and the Raspberry Pi firmware. We also do not recommend using the flatpak, since it can have permissions trouble when flashing the disk.

Reviewer: ```suggestion
- Install [Raspberry Pi Imager](https://www.raspberrypi.com/software/). You can grab this from the app store on Ultramarine. If you're on Windows or Mac you can [download it from here](https://www.raspberrypi.com/software/), or use it [directly in the Raspberry Pi firmware](https://www.raspberrypi.com/documentation/computers/getting-started.html#install-over-the-network). 
```

- [rpi-update](https://github.com/raspberrypi/rpi-update), a script to update Raspberry Pi firmware.
- [Picotool](https://github.com/raspberrypi/picotool), a tool for working with RP2040/RP2350 devices.

> We also have the [Raspberry Pi Control Center](https://github.com/raspberrypi-ui/rpcc) application, and it's various plugins:

Reviewer: ```suggestion
We also have the [Raspberry Pi Control Center](https://github.com/raspberrypi-ui/rpcc) application, and its various plugins:
```
